packages/core/components/ExternalInput/index.tsx

```diff
@@ -244,7 +244,7 @@ export const ExternalInput = ({
                         style={{ whiteSpace: "nowrap" }}
                         className={getClassNameModal("tr")}
                         onClick={() => {
-                          onChange(mapProp(data[i]));
+                          onChange(mapProp(data[i],filters));
```

Comment:
This needs to be formatted, otherwise it will fail linting. You can run `npm run format` to address it.